api/all/src/main/java/io/opentelemetry/api/internal/Contract.java

```diff
@@ -89,9 +89,9 @@
  * verifying nullness. We copy the annotation to avoid an external dependency.
  */
 @Documented
-@Retention(RetentionPolicy.SOURCE)
+@Retention(RetentionPolicy.CLASS)
```

Comment:
@jkwatson It turns out this needs to be CLASS so that other artifacts that use an annotated method (so usages of `StringUtils.isNullOrEmpty` outside of the api/all project) can be verified. I know you had a concern of us accidentally adding this to a non-internal API, so for now I've just made it package private since we only have one usage (well technically I wanted to use it in one other place but the workaround is fine)

Reply:
ok